guardian/typerighter
# Set rowcount to data array length

## What does this change?

The total returned by the search endpoint is sometimes greater than the number of rows for some queries in PROD. Not clear why this is but this change will allow editing of rules while we work on the issue with the SQL query



## How to test

Check out locally, point at PROD, then try searching for `child se` - there should be no error. Trying the same in prod results in an unhandled exceptio in the client 


## How can we measure success?

THe app now working for those search terms

## Have we considered potential risks?

This masks the error - we should not move on but fix the underlying issue with the response from the SQL query
